src/lib/slack/commands/ThingsBotCommand.js

```diff
@@ -71,15 +71,12 @@ const ThingsBotCommand = {
             };
         })();
     }
-}
+};
 
 async function getDeviceList(installedSmartAppId) {
-    const [ contextError, smartappContext ] = await smartappService.getSmartAppContext(installedSmartAppId);
-    if (contextError) { return; }
+    const smartAppContext = await smartAppService.getSmartAppContext(installedSmartAppId);
+    const deviceList = await smartAppService.getDeviceList(smartAppContext);
```

Comment:
By stripping the error handling here and throwing an error for problems getting context in `smartAppService`, aren't we just going to leave the user hanging with no error response? Or is that error going to get caught somewhere so the user gets updated that there was a problem?

Reply:
Ya I think I have to visit this a bit, I was thinking that if we wanted an upstream call to handle errors they should try and catch these method invocations explicitly.

Again just a stylistic change here that felt more familiar to me